src/Endpoints/Zones.php

```diff
@@ -27,18 +27,19 @@ public function __construct(Adapter $adapter)
      *
      * @param string $name
      * @param bool $jumpStart
-     * @param string $organizationID
+     * @param string $accountId
      * @return \stdClass
      */
-    public function addZone(string $name, bool $jumpStart = false, string $organizationID = ''): \stdClass
+    public function addZone(string $name, bool $jumpStart = false, string $accountId = ''): \stdClass
     {
         $options = [
             'name' => $name,
             'jump_start' => $jumpStart
         ];
 
-        if (!empty($organizationID)) {
-            $options['organization'] = ['id' => $organizationID];
+        if (!empty($accountId)) {
+            $options['organization'] = ['id' => $accountId];
```

Comment:
we only want one or the other here; we don't want both. just go with accountID for now.

Reply:
I've removed this